docs/templates/e-commerce.rst

```diff
@@ -0,0 +1,24 @@
+.. _e-commerce:
+
+===========================================
+E-commerce spider template (``e-commerce``)
+===========================================
+
+Basic use
+=========
+
+.. code-block:: shell
+
+    scrapy crawl e-commerce -a url="https://books.toscrape.com"
+
+Parameters
+==========
+
+.. autopydantic_model:: zyte_spider_templates.spiders.ecommerce.EcommerceSpiderParams
+    :inherited-members: BaseModel
+
+.. autoenum:: zyte_spider_templates.spiders.ecommerce.EcommerceCrawlStrategy
+
+.. autoenum:: zyte_spider_templates.spiders.ecommerce.ExtractFrom
```

Comment:
We need a more descriptive text for this one. The same case with **Geolocation**.

![image](https://github.com/zytedata/zyte-spider-templates/assets/3449761/d6ef4099-ae1a-4527-a567-68e2f9492fc1)

Reply:
In my rendering, I do not get “An enumeration.”, but rather nothing at all. Which I think is fine given the parameter descriptions themselves at the top. No strong opinion, though.